ui/page/root/home_page.go

```diff
@@ -233,6 +233,76 @@ func (hp *HomePage) initDEX() {
 		dexPassEditor := hp.Theme.EditorPassword(new(widget.Editor), values.String(values.StrDexPassword))
 		dexPassEditor.Editor.SingleLine, dexPassEditor.IsRequired = true, true
 
+		showWalletsToSyncModal := func() {
```

Comment:
New thought: request wallet sync before unlocking the dex client. It'd be better for the dex client to find wallets already syncing by the time the dex is logged in.

Reply:
ok